packages/pcd/pod-ticket-pcd/src/schema.ts

```diff
@@ -13,12 +13,28 @@ export const TicketDataSchema = z.object({
   productId: z.string().uuid(),
   timestampConsumed: z.number().int().nonnegative(),
   timestampSigned: z.number().int().nonnegative(),
-  attendeeSemaphoreId: z.string().refine(canBeBigInt).transform(cryptographic),
+  /**
+   * V3 semaphore commitment.
+   * Optional, used only by legacy v3 tickets.
+   */
+  attendeeSemaphoreId: z
+    .string()
+    .refine(canBeBigInt)
+    .optional()
+    // important that the transform comes last, otherwise
+    // `dataToPodEntries` will not work
+    .transform(cryptographic),
+  attendeeSemaphoreV4Id: z
+    .string()
+    .refine(canBeBigInt)
+    .optional()
```

Comment:
needs to be optional so that deserialization is backwards-compatible w/ old pod tickets

Reply:
not a huge fan of the naming. thoughts on the following?
- change name of `attendeeSemaphoreV4Id` to `owner` to match convention with other PODs
- remove the existing `attendeeSemaphoreId` field and don't allow backward compatibility. basically, we don't care about PODTicketPCD that have `attendeeSemaphoreId` -- you can't do anything with them.

Reply:
> change name of attendeeSemaphoreV4Id to owner to match convention with other PODs

done

> remove the existing attendeeSemaphoreId field and don't allow backward compatibility. basically, we don't care about PODTicketPCD that have attendeeSemaphoreId -- you can't do anything with them.

for peace of mind I'm going to keep this field.